### ISSUE TITLE
allow removing variables from environment by just clicking on variables in env pane

### ISSUE THREAD

**Issue by sanjmeh** [rstudio/rstudio#11267]
Removing large variables from memory is regularly required to 
a) _keep the memory from bloating up_ and
b) _removing clutter_. 
One way is **restarting** R session without auto loading of memory (donot save in `.RData`). But a lot of times I donot want to lose some major variables after R is restarted. And if R is configured to reload all variables the purpose is defeated of restarting.  
The command line way to remove variables is a bit cumbersome.
It goes like this.
```
# check all variables with a pattern
ls(pattern = "dt")
# chose the ones you want to retain and then eitehr using a script or just typing the names of variables we can remove the undesired variables e.g. - 
rm(addon_dt, blank_edt, blank_elm_dt, blank_fdt, clean_gapsdt, combine_fdts)
```
This is clumsy and error prone.

How about giving a mouse click feature to remove variable from environment pane.

The advantage is we can see the data in the variable, along side other variables, so it is easiest to chose here. 

See a toy wireframe. 
![Screen Shot 2022-05-22 at 7 48 11 PM](https://user-images.githubusercontent.com/3091624/169700726-0cf75d0a-82b7-42dd-8dac-54b4a011225f.png)

Basically the user gets to choose which variables they want to remove from the list which already shows the contents of each variable. So it is safest and fastest to allow their removal directly inside the environment pane.

**Comment by ronblum** [comment]
@sanjmeh Thank you for the suggestion! We'll review it as we continue to develop and update RStudio.

**Comment by jmcphers** [comment]
Good news, we already have this! Click the "list" selector at the top and change to grid view. In grid view, a checkbox will appear next to each item; you can check the ones you want to remove and then click the "broom" icon to remove them. You can also sort by size if you want to remove the biggest contributors to your working set.

<img width="558" alt="image" src="https://user-images.githubusercontent.com/470418/169860703-a84b684a-deb5-4b5b-aae1-0a391a9563b3.png">

**Comment by ronblum** [comment]
TIL how to use a product I've been testing for over four years. 😄

**Comment by sanjmeh** [comment]
Unbelievable  @ronblum ! I missed this feature even while I have been using rstudio for 4 years !  🥇

**Comment by jmcphers** [comment]
It's very hard to find even if you're looking for it! It was introduced because we couldn't figure out how to make the list view useful for sorting and cleaning w/o adding a lot of clutter and complexity (plus sorting and grouping get complicated). Next time we revisit the Environment pane we'll try to find a better way to address this!
